src/Admin.php

```diff
@@ -136,12 +136,22 @@ public static function post_action_label($post_id) {
       $discount_percentage = 0;
     }
 
+    $formatted_regular_price = wc_price($regular_price, ['price_format' => '%1$s&nbsp;%2$s']);
+    // Replace ",00" with ",-".
+    $formatted_regular_price = preg_replace('/\,0++/', ',-', $formatted_regular_price);
```

Comment:
We use '@' as separator in regex to avoid having to scape '\' chars.

Reply:
Typography: The hyphen should not be a regular hyphen, but a dash to signify that something has been omitted. The idea is that the dash is as long/wide as the two zeros that were omitted. This is the case for the em-dash:

64,00
64,— (emdash)
64,– (endash)
64,- (hyphen)

Technically, usage of the dash for prices is only foreseen for price tables (where many prices appear below and next to each other) so as to reduce visual clutter.  For price labels that are standing alone, the flat price without decimals is more readable and more clear.  But if our customers want to display them with a dash for regular price labels, we can certainly try to do so.

See also https://www.typografie.info/3/topic/16761-regeln-bei-%E2%82%AC-schreibweise/
https://de.wikipedia.org/wiki/Geviertstrich
https://en.wikipedia.org/wiki/Dash#Em_dash

Please use the Unicode character in the code, not the HTML entity.

Reply:
> We use '@' as separator in regex to avoid having to scape '' chars.

@juanlopez4691 can you give me an example in order to learn something from it please? I haven't found out anything in research about '@' as seperator in regex. I'm not so well versed with regex yet.

Reply:
@sun Thanks for the typography details – there's always something new to learn. :)
I have used the endash in the end, because an emdash looked way too long. Hope this works for you. ;) 
This is how it looks now:
<img width="1367" alt="document__45__pdf" src="https://user-images.githubusercontent.com/6773669/68988505-fb910f00-0838-11ea-8e6a-31f4fa77cc4a.png">